From aab02e1c386fa2ab761aea583b59ce72824f70c4 Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@siol.net>
Date: Thu, 8 Mar 2018 17:21:29 +0100
Subject: [PATCH 78/82] drm/sun4i: Enable auto_cts quirk for DW HDMI

Allwinner SoCs with DW HDMI don't output any sound if auto CTS option is
not enabled.

Enable it.

Signed-off-by: Jernej Skrabec <jernej.skrabec@siol.net>
---
 drivers/gpu/drm/sun4i/sun8i_dw_hdmi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/sun4i/sun8i_dw_hdmi.c b/drivers/gpu/drm/sun4i/sun8i_dw_hdmi.c
index 31875b636434..9aaec57f7441 100644
--- a/drivers/gpu/drm/sun4i/sun8i_dw_hdmi.c
+++ b/drivers/gpu/drm/sun4i/sun8i_dw_hdmi.c
@@ -156,6 +156,7 @@ static int sun8i_dw_hdmi_bind(struct device *dev, struct device *master,
 
 	sun8i_hdmi_phy_init(hdmi->phy);
 
+	plat_data->auto_cts = true;
 	plat_data->mode_valid = &sun8i_dw_hdmi_mode_valid;
 	plat_data->phy_ops = sun8i_hdmi_phy_get_ops();
 	plat_data->phy_name = "sun8i_dw_hdmi_phy";
-- 
2.20.1

